diff --git a/picsimlab/Makefile.Common b/picsimlab/Makefile.Common
index 93c9f21..d5f53e1 100644
--- a/picsimlab/Makefile.Common
+++ b/picsimlab/Makefile.Common
@@ -1,19 +1,19 @@
 CPPFLAGS=  -fPIC -g -O2 -I. -I.. -I../cmd.src -I../sim.src -I../gui.src 
 
-s51_OBJECTS_SHARED_ =$(shell cd ../s51.src/; $(MAKE)  -f Makefile  -f ../picsimlab/print.mak print-OBJECTS_SHARED)
-stm8_OBJECTS_SHARED_=$(shell cd ../stm8.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-OBJECTS_SHARED)
-z80_OBJECTS_SHARED_ =$(shell cd ../z80.src/; $(MAKE)  -f Makefile  -f ../picsimlab/print.mak print-OBJECTS_SHARED)
+s51_OBJECTS_SHARED_ =$(shell cd ../s51.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-OBJECTS_SHARED)
+stm8_OBJECTS_SHARED_=$(shell cd ../stm8.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-OBJECTS_SHARED)
+z80_OBJECTS_SHARED_ =$(shell cd ../z80.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-OBJECTS_SHARED)
 
-GUI_OBJECTS_  =$(shell cd ../gui.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-OBJECTS)
-GUI_OBJECTS_ +=$(shell cd ../gui.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-UCSIM_OBJECTS)
-GUI_OBJECTS_ +=$(shell cd ../gui.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-TOOL_OBJECTS)
+GUI_OBJECTS_  =$(shell cd ../gui.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-OBJECTS)
+GUI_OBJECTS_ +=$(shell cd ../gui.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-UCSIM_OBJECTS)
+GUI_OBJECTS_ +=$(shell cd ../gui.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-TOOL_OBJECTS)
 
-CMD_OBJECTS_  =$(shell cd ../cmd.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-OBJECTS)
-CMD_OBJECTS_ +=$(shell cd ../cmd.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-LOCAL_OBJECTS)
+CMD_OBJECTS_  =$(shell cd ../cmd.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-OBJECTS)
+CMD_OBJECTS_ +=$(shell cd ../cmd.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-LOCAL_OBJECTS)
 
-SIM_OBJECTS_  =$(shell cd ../sim.src/; $(MAKE) -f Makefile  -f ../picsimlab/print.mak print-OBJECTS)
+SIM_OBJECTS_  =$(shell cd ../sim.src/; $(MAKE) -s -f Makefile  -f ../picsimlab/print.mak print-OBJECTS)
 
-MAIN_OBJECTS_  =$(shell cd ..; $(MAKE) -f main.mk  -f picsimlab/print.mak print-OBJECTS)
+MAIN_OBJECTS_  =$(shell cd ..; $(MAKE) -s -f main.mk  -f picsimlab/print.mak print-OBJECTS)
 
 
 
